app/src/main/java/org/astraea/topic/ReplicaSyncingMonitor.java

```diff
@@ -225,7 +229,7 @@ public String progressBar() {
     }
 
     public double dataRatePerSec() {
-      return (double) (currentSize - previousSize) / ((interval.toNanos() / 1e9));
+      return dataRate.dataRate(DataUnit.Byte, ChronoUnit.SECONDS).doubleValue();
```

Comment:
we call `dataRate.dataRate` in the class `dataRate`. I guess there are something nested with each other.